util/chplenv/chpl_libfabric.py

```diff
@@ -12,7 +13,10 @@ def get():
         libfabric_val = overrides.get('CHPL_LIBFABRIC')
         platform_val = chpl_platform.get('target')
         if not libfabric_val:
-            if platform_val == 'hpe-cray-ex':
+            exists, returncode = try_run_command(['pkg-config',
+                                                  '--exists',
+                                                  'libfabric'])[0:2]
```

Comment:
What is the `[0:2]` giving you here? Just getting rid of the output? Typically this would be more like:


```python
            exists, returncode, _ = try_run_command(['pkg-config',
                                                  '--exists',
                                                  'libfabric'])
```

Reply:
I looked at some discussion about this, notably an entry on [stackoverflow](https://stackoverflow.com/questions/431866/ignore-python-multiple-return-value), and decided I liked this better.